server/app.js

```diff
@@ -1,17 +1,29 @@
 import express from 'express';
-
-const app = express();
+import bodyParser from 'body-parser';
+import cookieParser from 'cookie-parser';
+import cors from 'cors';
 
 //Import all Routes
 import userRoutes from './routes/userRoutes';
-import postRoutes from './routes/userRoutes';
-import notificationRoutes from './routes/userRoutes';
-import messageRoutes from './routes/userRoutes';
+import postRoutes from './routes/postRoutes';
+import notificationRoutes from './routes/notificationRoutes';
+import messageRoutes from './routes/messageRoutes';
+
+const app = express();
+
+//Middlewares
+app.use(bodyParser.json());
```

Comment:
`bodyParser` is unnecessary here...simply add `app.use(express.json())` [Express JS— body-parser and why may not need it](https://medium.com/@mmajdanski/express-body-parser-and-why-may-not-need-it-335803cd048c)

Reply:
You're right. I'm not aware of it. correcting asap. 👍